<Type Name="ForeignKeyReferenceAlreadyHasValueException" FullName="System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException">
  <Metadata>
    <Meta Name="ms.openlocfilehash" Value="2b95b808a6054194ebe2cd4a7b004ff5e6852b77" />
    <Meta Name="ms.sourcegitcommit" Value="9dda17222b9f7d3edf130133bfb1370d5b410a4b" />
    <Meta Name="ms.translationtype" Value="MT" />
    <Meta Name="ms.contentlocale" Value="pl-PL" />
    <Meta Name="ms.lasthandoff" Value="09/22/2018" />
    <Meta Name="ms.locfileid" Value="46668686" />
  </Metadata>
  <TypeSignature Language="C#" Value="public class ForeignKeyReferenceAlreadyHasValueException : InvalidOperationException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi beforefieldinit ForeignKeyReferenceAlreadyHasValueException extends System.InvalidOperationException" />
  <TypeSignature Language="DocId" Value="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" />
  <TypeSignature Language="VB.NET" Value="Public Class ForeignKeyReferenceAlreadyHasValueException&#xA;Inherits InvalidOperationException" />
  <TypeSignature Language="C++ CLI" Value="public ref class ForeignKeyReferenceAlreadyHasValueException : InvalidOperationException" />
  <TypeSignature Language="F#" Value="type ForeignKeyReferenceAlreadyHasValueException = class&#xA;    inherit InvalidOperationException" />
  <AssemblyInfo>
    <AssemblyName>System.Data.Linq</AssemblyName>
    <AssemblyVersion>3.5.0.0</AssemblyVersion>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.InvalidOperationException</BaseTypeName>
  </Base>
  <Interfaces />
  <Docs>
    <summary>
      <span data-ttu-id="d35d9-101">Reprezentuje błędy, które występują, gdy podejmowana jest próba zmiany klucza obcego, gdy jednostka jest już załadowany.</span>
      <span class="sxs-lookup">
        <span data-stu-id="d35d9-101">Represents errors that occur when an attempt is made to change a foreign key when the entity is already loaded.</span>
      </span>
    </summary>
    <remarks>To be added.</remarks>
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>System.Data.Linq</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary>
          <span data-ttu-id="d35d9-102">Inicjuje nowe wystąpienie klasy <see cref="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" /> klasy.</span>
          <span class="sxs-lookup">
            <span data-stu-id="d35d9-102">Initializes a new instance of the <see cref="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" /> class.</span>
          </span>
        </summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public ForeignKeyReferenceAlreadyHasValueException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; ForeignKeyReferenceAlreadyHasValueException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Data.Linq</AssemblyName>
        <AssemblyVersion>3.5.0.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Attributes>
        <Attribute FrameworkAlternate="netframework-4.0">
          <AttributeName>System.Runtime.TargetedPatchingOptOut("Performance critical to inline this type of method across NGen image boundaries")</AttributeName>
        </Attribute>
      </Attributes>
      <Parameters />
      <Docs>
        <summary>
          <span data-ttu-id="d35d9-103">Inicjuje nowe wystąpienie klasy <see cref="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" /> klasie z atrybutem dostarczane przez system komunikatu, który opisuje błąd.</span>
          <span class="sxs-lookup">
            <span data-stu-id="d35d9-103">Initializes a new instance of the <see cref="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" /> class with a system-supplied message that describes the error.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="d35d9-104">Ten konstruktor inicjuje <xref:System.Exception.Message%2A> właściwości nowego wystąpienia dostarczane przez system komunikatu, który opisuje błąd, takie jak "DefaultMessageDisplayedByParameterlessCtorWriterMustSupply".</span><span class="sxs-lookup"><span data-stu-id="d35d9-104">This constructor initializes the <xref:System.Exception.Message%2A> property of the new instance to a system-supplied message that describes the error, such as "DefaultMessageDisplayedByParameterlessCtorWriterMustSupply".</span></span> <span data-ttu-id="d35d9-105">Ten komunikat uwzględnia bieżącą kulturę systemu.</span><span class="sxs-lookup"><span data-stu-id="d35d9-105">This message takes into account the current system culture.</span></span>  
  
 <span data-ttu-id="d35d9-106">W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.InsufficientMemoryException>.</span><span class="sxs-lookup"><span data-stu-id="d35d9-106">The following table shows the initial property values for an instance of <xref:System.InsufficientMemoryException>.</span></span>  
  
|<span data-ttu-id="d35d9-107">Właściwość</span><span class="sxs-lookup"><span data-stu-id="d35d9-107">Property</span></span>|<span data-ttu-id="d35d9-108">Wartość</span><span class="sxs-lookup"><span data-stu-id="d35d9-108">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="d35d9-109">`null`.</span><span class="sxs-lookup"><span data-stu-id="d35d9-109">`null`.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="d35d9-110">Zlokalizowany ciąg komunikatu o błędzie.</span><span class="sxs-lookup"><span data-stu-id="d35d9-110">The localized error message string.</span></span>|  
  
 ]]></format>
        </remarks>
        <related type="Article" href="~/docs/standard/exceptions/index.md">
          <span data-ttu-id="d35d9-111">Obsługa i zgłaszanie wyjątków</span>
          <span class="sxs-lookup">
            <span data-stu-id="d35d9-111">Handling and Throwing Exceptions</span>
          </span>
        </related>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public ForeignKeyReferenceAlreadyHasValueException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; ForeignKeyReferenceAlreadyHasValueException(System::String ^ message);" />
      <MemberSignature Language="F#" Value="new System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException : string -&gt; System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" Usage="new System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException message" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Data.Linq</AssemblyName>
        <AssemblyVersion>3.5.0.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Attributes>
        <Attribute FrameworkAlternate="netframework-4.0">
          <AttributeName>System.Runtime.TargetedPatchingOptOut("Performance critical to inline this type of method across NGen image boundaries")</AttributeName>
        </Attribute>
      </Attributes>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="d35d9-112">Komunikat, który opisuje wyjątek.</span>
          <span class="sxs-lookup">
            <span data-stu-id="d35d9-112">The message that describes the exception.</span>
          </span>
          <span data-ttu-id="d35d9-113">Obiekt wywołujący tego konstruktora jest wymagany w celu zagwarantowania, że ten ciąg został zlokalizowany w bieżącej kulturze systemu.</span>
          <span class="sxs-lookup">
            <span data-stu-id="d35d9-113">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="d35d9-114">Inicjuje nowe wystąpienie klasy <see cref="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" /> klasy przy użyciu określonego komunikatu, który opisuje błąd.</span>
          <span class="sxs-lookup">
            <span data-stu-id="d35d9-114">Initializes a new instance of the <see cref="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" /> class with a specified message that describes the error.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="d35d9-115">W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException>.</span><span class="sxs-lookup"><span data-stu-id="d35d9-115">The following table shows the initial property values for an instance of <xref:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException>.</span></span>  
  
|<span data-ttu-id="d35d9-116">Właściwość</span><span class="sxs-lookup"><span data-stu-id="d35d9-116">Property</span></span>|<span data-ttu-id="d35d9-117">Wartość</span><span class="sxs-lookup"><span data-stu-id="d35d9-117">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="d35d9-118">`null`.</span><span class="sxs-lookup"><span data-stu-id="d35d9-118">`null`.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="d35d9-119">Ciąg komunikatu o błędzie określony w `message`.</span><span class="sxs-lookup"><span data-stu-id="d35d9-119">The error message string specified in `message`.</span></span>|  
  
 ]]></format>
        </remarks>
        <related type="Article" href="~/docs/standard/exceptions/index.md">
          <span data-ttu-id="d35d9-120">Obsługa i zgłaszanie wyjątków</span>
          <span class="sxs-lookup">
            <span data-stu-id="d35d9-120">Handling and Throwing Exceptions</span>
          </span>
        </related>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public ForeignKeyReferenceAlreadyHasValueException (string message, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; ForeignKeyReferenceAlreadyHasValueException(System::String ^ message, Exception ^ innerException);" />
      <MemberSignature Language="F#" Value="new System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException : string * Exception -&gt; System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" Usage="new System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException (message, innerException)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Data.Linq</AssemblyName>
        <AssemblyVersion>3.5.0.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Attributes>
        <Attribute FrameworkAlternate="netframework-4.0">
          <AttributeName>System.Runtime.TargetedPatchingOptOut("Performance critical to inline this type of method across NGen image boundaries")</AttributeName>
        </Attribute>
      </Attributes>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="d35d9-121">Komunikat, który opisuje wyjątek.</span>
          <span class="sxs-lookup">
            <span data-stu-id="d35d9-121">The message that describes the exception.</span>
          </span>
          <span data-ttu-id="d35d9-122">Obiekt wywołujący tego konstruktora jest wymagany w celu zagwarantowania, że ten ciąg został zlokalizowany w bieżącej kulturze systemu.</span>
          <span class="sxs-lookup">
            <span data-stu-id="d35d9-122">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span>
          </span>
        </param>
        <param name="innerException">
          <span data-ttu-id="d35d9-123">Wyjątek, który jest przyczyną bieżącego wyjątku.</span>
          <span class="sxs-lookup">
            <span data-stu-id="d35d9-123">The exception that is the cause of the current exception.</span>
          </span>
          <span data-ttu-id="d35d9-124">Jeśli <c>innerException</c> parametr nie jest <see langword="null" />, bieżący wyjątek jest zgłaszany w <see langword="catch" /> blok, który obsługuje wyjątek wewnętrzny.</span>
          <span class="sxs-lookup">
            <span data-stu-id="d35d9-124">If the <c>innerException</c> parameter is not <see langword="null" />, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="d35d9-125">Inicjuje nowe wystąpienie klasy <see cref="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" /> klasy przy użyciu określonego komunikatu o błędzie i odwołania do wewnętrznego wyjątku, który jest przyczyną tego wyjątku.</span>
          <span class="sxs-lookup">
            <span data-stu-id="d35d9-125">Initializes a new instance of the <see cref="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" /> class with a specified error message and a reference to the inner exception that is the cause of this exception.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="d35d9-126">Wyjątek generowany jako bezpośredni wynik poprzedniego wyjątku powinien zawierać odwołanie do poprzedniego wyjątku we <xref:System.Exception.InnerException%2A> właściwości.</span><span class="sxs-lookup"><span data-stu-id="d35d9-126">An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="d35d9-127"><xref:System.Exception.InnerException%2A> Właściwość zwraca taką samą wartość, która jest przekazywana do konstruktora, lub `null` Jeśli <xref:System.Exception.InnerException%2A> nie dostarcza wartości wewnętrznego wyjątku do konstruktora.</span><span class="sxs-lookup"><span data-stu-id="d35d9-127">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.</span></span>  
  
 <span data-ttu-id="d35d9-128">W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException>.</span><span class="sxs-lookup"><span data-stu-id="d35d9-128">The following table shows the initial property values for an instance of <xref:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException>.</span></span>  
  
|<span data-ttu-id="d35d9-129">Właściwość</span><span class="sxs-lookup"><span data-stu-id="d35d9-129">Property</span></span>|<span data-ttu-id="d35d9-130">Wartość</span><span class="sxs-lookup"><span data-stu-id="d35d9-130">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="d35d9-131">`null`.</span><span class="sxs-lookup"><span data-stu-id="d35d9-131">`null`.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="d35d9-132">Ciąg komunikatu o błędzie określony w `message`.</span><span class="sxs-lookup"><span data-stu-id="d35d9-132">The error message string specified in `message`.</span></span>|  
  
 ]]></format>
        </remarks>
        <related type="Article" href="~/docs/standard/exceptions/index.md">
          <span data-ttu-id="d35d9-133">Obsługa i zgłaszanie wyjątków</span>
          <span class="sxs-lookup">
            <span data-stu-id="d35d9-133">Handling and Throwing Exceptions</span>
          </span>
        </related>
      </Docs>
    </Member>
  </Members>
</Type>